src/stories/Library/autosuggest-text/AutosuggestText.tsx

```diff
@@ -1,28 +1,24 @@
 export type AutosuggestTextProps = {
-  items: string[];
-  categoryText: string;
+  textSuggestions: string[];
+  categoryText?: string;
 };
 
 export const AutosuggestText = (props: AutosuggestTextProps) => {
-  const { items, categoryText } = props;
+  const { textSuggestions, categoryText } = props;
   return (
-    <ul className="autosuggest pb-16">
-      <li>
-        <ul>
-          {items.map((item) => {
-            return (
-              <li className="autosuggest__text text-body-medium-regular px-24">
-                {item}
-                {categoryText && (
-                  <div className="boxed-text text-tags noselect ml-8">
-                    {categoryText}
-                  </div>
-                )}
-              </li>
-            );
-          })}
-        </ul>
-      </li>
-    </ul>
+    <>
+      {textSuggestions.map((item) => {
+        return (
+          <li className="autosuggest__text-item text-body-medium-regular px-24">
+            <p className="autosuggest__text text-body-medium-regular">{item}</p>
+            {categoryText && (
+              <div className="boxed-text text-tags noselect ml-8">
+                {categoryText}
```

Comment:
This might seem like an oversigt, but is actually very intentional. 
Without inner <p>:
![image](https://user-images.githubusercontent.com/28546954/226669906-3359478d-318f-41ee-bcf7-fc0278fec125.png)

With inner <p>:
![image](https://user-images.githubusercontent.com/28546954/226669813-2a0b6037-d187-48b3-8dbc-86c0935691f3.png)